test/fixtures/fake_subscription_registration.json

```diff
@@ -0,0 +1,91 @@
+{
+    "id": "inv_JA7OELdAoxbzk7",
+    "entity": "invoice",
+    "receipt": "Receipt No. #12",
+    "invoice_number": "Receipt No. #12",
+    "customer_id": "cust_DzYEzfJLV03rkp",
+    "customer_details": {
+      "id": "cust_DzYEzfJLV03rkp",
+      "name": "Gaurav Kumar",
+      "email": "gaurav.kumar@example.com",
+      "contact": "9123456780",
+      "gstin": null,
+      "billing_address": null,
+      "shipping_address": null,
+      "customer_name": "Gaurav Kumar",
+      "customer_email": "gaurav.kumar@example.com",
+      "customer_contact": "9123456780"
+    },
+    "order_id": "order_JA7OEMWy90eSP6",
+    "line_items": [
+  
+    ],
+    "payment_id": null,
+    "status": "issued",
+    "expire_by": 1648101450,
+    "issued_at": 1647948473,
+    "paid_at": null,
+    "cancelled_at": null,
+    "expired_at": null,
+    "sms_status": "pending",
+    "email_status": "pending",
+    "date": 1647948473,
+    "terms": null,
+    "partial_payment": false,
+    "gross_amount": 0,
+    "tax_amount": 0,
+    "taxable_amount": 0,
+    "amount": 0,
+    "amount_paid": 0,
+    "amount_due": 0,
+    "currency": "INR",
+    "currency_symbol": "₹",
+    "description": "12 p.m. Meals",
+    "notes": {
+      "note_key 1": "Beam me up Scotty",
+      "note_key 2": "Tea. Earl Gray. Hot."
+    },
+    "comment": null,
+    "short_url": "https://rzp.io/i/cXyLlMxVLB",
+    "view_less": true,
+    "billing_start": null,
+    "billing_end": null,
+    "type": "link",
+    "group_taxes_discounts": false,
+    "created_at": 1647948473,
+    "idempotency_key": null,
+    "auth_link_status": "issued",
+    "token": {
+      "method": "nach",
+      "notes": {
+        "note_key 1": "Beam me up Scotty",
+        "note_key 2": "Tea. Earl Gray. Hot."
+      },
+      "recurring_status": null,
+      "failure_reason": null,
+      "currency": "INR",
+      "max_amount": 50000,
+      "auth_type": "physical",
+      "expire_at": 1648101450,
+      "nach": {
+        "create_form": true,
+        "form_reference1": "Recurring Payment for Gaurav Kumar",
+        "form_reference2": "Method Paper NACH",
+        "prefilled_form": "https://rzp.io/i/VvObPb0ht",
+        "prefilled_form_transient": "https://rzp.io/i/jlERKqU",
+        "upload_form_url": "https://rzp.io/i/cXyLlMxVLB",
+        "description": "12 p.m. Meals"
+      },
+      "bank_account": {
+        "ifsc": "HDFC0001233",
+        "bank_name": "HDFC Bank",
+        "name": "Gaurav Kumar",
+        "account_number": "11214311215411",
+        "account_type": "savings",
+        "beneficiary_email": "gaurav.kumar@example.com",
+        "beneficiary_mobile": "9123456780"
+      },
+      "first_payment_amount": 0
+    },
+    "nach_form_url": "https://rzp.io/i/jlERKqU"
+  }
```

Comment:
fix indentation and add new line at end of file

Reply:
eol fixed